geocoder.js

```diff
@@ -422,7 +422,8 @@ var geocoder = {
       var lng = lineObj[GEONAMES_COLUMNS[latitudeIndex]];
       var lat = lineObj[GEONAMES_COLUMNS[longitudeIndex]];
       //dont add lineObj without lat/lng pair
-      if(lng !== null && lng !== undefined && !isNaN(lng) && lat !== null && lat !== undefined && !isNaN(lat)) {
+      if(lng !== null && lng !== undefined && !isNaN(lng) && lat !== null
```

Comment:
Thanks for your patience. The code should follow [Google's style guide](https://google.github.io/styleguide/javascriptguide.xml). To automate style linting, you can use [jscs](http://jscs.info/overview) with the [google preset](http://jscs.info/overview#cli).